app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -204,7 +208,7 @@ class SuggestedEditsTasksFragment : Fragment() {
                 }
                 .observeOn(AndroidSchedulers.mainThread())
                 .doAfterTerminate {
-                    if (blockInfo != null) {
+                    if (blockMessage != null) {
```

Comment:
Would it be better if we use `!blockMessage.isNullOrEmpty()`?

Reply:
correct!